finished headers for all programs

shuuvi opened the pull request:
![image](https://github.com/user-attachments/assets/b971cccd-7b12-43bb-89eb-2b25888a0af7)
![image](https://github.com/user-attachments/assets/5903f671-cb9f-4345-a0d9-1a892e1bffb7)
![image](https://github.com/user-attachments/assets/087b5192-cc97-4691-ad28-ae2381476c43)
![image](https://github.com/user-attachments/assets/c0cb3fbd-1413-46d3-99b7-a376b945c123)
![image](https://github.com/user-attachments/assets/bcfc3fb4-0c6c-4592-8d8a-da0144c2c7f0)
I created the headers for all of the programs and added the svgs for each logo. :D

shuuvi committed:
created all of the headers for the programs

shuuvi committed:
Merge branch 'dev' of https://github.com/acm-ucr/acm-atlas into shuuvi/boardCard

shuuvi committed:
Merge 770c69cedabe6a559f0a5663b58916e62e60bd2c into 5368a0ce5ee4891b706faed5964eedb000c7ceb5

Commit:
convert images to webp

stanleylew5 committed:
Merge pull request #940 from acm-ucr/acm-ucr/images_f4e5309158e4431e

Update Images to Webp

stanleylew5 commented on src/components/programs/spark/header.tsx:
```diff
@@ -1,8 +1,19 @@
+        ACM Spark is a 10 week program that focuses on static front-end web
+        development for campus organizations / companies.
+      </p>
+      <Image className="absolute top-15 right-90" src={Spark} alt="Spark" />
```
use % values for top and right

stanleylew5 replied:
do this for all the rest

stanleylew5 commented on src/components/programs/spark/header.tsx:
```diff
@@ -1,8 +1,19 @@
-    </p>
+    <div className="text-acm-gray-500 relative p-10 text-2xl font-semibold">
+      <div className="text-center">
+        <p className="flex justify-center p-7 text-5xl font-bold">SPARK</p>
```
change p-7 to p-5

stanleylew5 commented on src/components/programs/spark/header.tsx:
```diff
@@ -1,8 +1,19 @@
+    <div className="text-acm-gray-500 relative p-10 text-2xl font-semibold">
+      <div className="text-center">
+        <p className="flex justify-center p-7 text-5xl font-bold">SPARK</p>
+        <p className="pb-7">Web Development</p>
```
```suggestion
        <p className="pb-5">Web Development</p>
```

stanleylew5 reviewed (changes requested):
![image](https://github.com/user-attachments/assets/74505c7d-ddf6-40fd-9f91-8c010a1cfb52)
all the pages have this glowy ellipse so include this in all of them too

shuuvi committed:
Merge branch 'dev' of https://github.com/acm-ucr/acm-atlas into shuuvi/boardCard

shuuvi committed:
moved the logos to to logo folder and created glowing ellipse

shuuvi committed:
moved the logos to to logo folder and created glowing ellipse

shuuvi committed:
changed the svg type in das header

shuuvi commented:
![image](https://github.com/user-attachments/assets/cab352f4-bbb5-403f-a1fd-dd1bbb9194d3)
![image](https://github.com/user-attachments/assets/07bd22ec-31a1-46fb-9798-9c55f5757550)
![image](https://github.com/user-attachments/assets/60c54770-c3af-492d-8d6f-0ace414cfe42)
![image](https://github.com/user-attachments/assets/5859dd98-b120-487e-8edc-64b05bcf3c31)
![image](https://github.com/user-attachments/assets/dd7f260b-031a-484e-94c6-cc228abf0fb2)
fixed and added requested changes for the glowing ellipse

stanleylew5 commented on public/DAS_2.webp:
delete this file

stanleylew5 commented on public/create1.webp:
delete this file

stanleylew5 commented on public/forge_1.webp:
delete this file

stanleylew5 commented on public/logos/bitbytes.svg:
just bitbyte without the s

stanleylew5 commented on public/spark_1.webp:
delete this file

stanleylew5 commented on src/components/programs/bitbyte/header.tsx:
```diff
@@ -1,7 +1,23 @@
-    <div className="flex flex-col items-center justify-center">
-      <p className="text-acm-gray-500 text-7xl font-bold">BIT/BYTE</p>
+    <div className="text-acm-gray-500 relative overflow-hidden p-10 text-2xl font-semibold">
+      <div className="bg-acm-green-100 absolute top-0 z-0 h-[60%] w-[30%] rounded-full opacity-50 blur-3xl" />
```
instead of divs have them as svgs

if you absolutely position them with top-0 and left-0 they should look something like this and there won't be any overlap with the text, do this for all the files
![image](https://github.com/user-attachments/assets/bb04944c-095c-4be9-89d4-f7b8535a29ba)

stanleylew5 replied:
```suggestion
```

stanleylew5 commented on src/components/programs/create/header.tsx:
```diff
@@ -1,8 +1,24 @@
+      <div className="text-center">
+        <p className="flex justify-center p-7 text-5xl font-bold">CREATE</p>
+        <p className="pb-7">UI/UX Design</p>
+      </div>
```
doesn't need the outer div, just add text-center to the second p tag, do this for all the other files

stanleylew5 commented on src/components/programs/bitbyte/header.tsx:
```diff
@@ -1,7 +1,23 @@
+    <div className="text-acm-gray-500 relative overflow-hidden p-10 text-2xl font-semibold">
+      <div className="bg-acm-green-100 absolute top-0 z-0 h-[60%] w-[30%] rounded-full opacity-50 blur-3xl" />
+      <div className="text-center">
+        <p className="flex justify-center p-5 text-5xl font-bold">BIT/BYTES</p>
```
```suggestion
        <p className="flex justify-center p-5 text-5xl font-bold">BIT/BYTE</p>
```

stanleylew5 commented on src/components/programs/bitbyte/header.tsx:
```diff
@@ -1,7 +1,23 @@
+      <Image
+        className="absolute top-[20%] right-[25%]"
+        src={BitBytes}
+        alt="BitBytes"
```
```suggestion
        src={BitByte}
        alt="BitByte"
```

stanleylew5 commented on src/components/programs/bitbyte/header.tsx:
```diff
@@ -1,7 +1,23 @@
+import Image from "next/image";
+import BitBytes from "@/public/logos/bitbytes.svg";
```
BitByte

stanleylew5 commented on src/components/programs/bitbyte/header.tsx:
```diff
@@ -1,7 +1,23 @@
   return (
-    <div className="flex flex-col items-center justify-center">
-      <p className="text-acm-gray-500 text-7xl font-bold">BIT/BYTE</p>
+    <div className="text-acm-gray-500 relative overflow-hidden p-10 text-2xl font-semibold">
```
```suggestion
    <div className="text-acm-gray-500 relative p-10 text-2xl font-semibold">
```

shuuvi committed:
Merge branch 'dev' of https://github.com/acm-ucr/acm-atlas into shuuvi/boardCard some thing?

shuuvi committed:
added the svgs for the ellipses and adjusted to changes

shuuvi commented:
the webstie looks the same as the previous image, and fixed requested changes, the padding for the first div is pb because when I do p-10 it cuts off the SVG even when its absolute.

stanleylew5 committed:
remove the letter s

stanleylew5 committed:
empty commit

stanleylew5 commented on src/components/programs/bitbyte/header.tsx:
wait sorry lowkey i just realized this can you make these all one reusable component and pass in props and just call it like programheader.tsx and put it within components/programs

shuuvi committed:
finished requested changes

shuuvi committed:
Merge branch 'shuuvi/boardCard' of https://github.com/acm-ucr/acm-atlas into shuuvi/boardCard commit errors

shuuvi commented:
I created a new file with the prop names, but didn't create the data file and left the original code in the headers as is.

stanleylew5 reviewed (changes requested):
instead of having a header.tsx file for create, das, forge, spark, bitbyte, just call the header programheader component and pass in the props in their respective page.tsx files

shuuvi committed:
called the props from coments and added them to page.tsx adn delted header files

shuuvi commented:
Deleted the header.tsx for all programs and did the requested changes and called props components into the page.tsx of each respective program :D
